src/pages/Mining/index.tsx

```diff
@@ -0,0 +1,99 @@
+import Card from "common/Card";
+import Row from "common/Row";
+import Calculator from "components/Calculator";
+import QualityPicker from "components/QualityPicker";
+import { FC, useContext, useEffect, useState } from "react";
+import ReactDropdown, { Option } from "react-dropdown";
+import { CalculatorContext } from "state/Calculator/Context";
+import {
+  OreNames,
+  xpPerOre,
+  respawnPerOre,
+  pickaxeTimes,
+  unlockables,
+} from "./data";
+
+const potionSpeeds = [1, 1.1, 1.2, 1.4, 1.8];
+
+const Mining: FC = () => {
+  const {
+    calculatorState: { itemMastery, masteryPool },
+  } = useContext(CalculatorContext);
+
+  const [potionLevel, setPotionLevel] = useState<number>(1);
+  const [timeReduction, setTimeReduction] = useState(1);
+  const [ore, setOre] = useState<OreNames>("Tin");
+  const [xps, setXps] = useState(0);
+
+  useEffect(() => {
+    setXps(
+      calculateXPperSecond(
+        ore,
+        3 * timeReduction,
+        Number(itemMastery[0]),
+        potionLevel,
+        Number(masteryPool)
+      )
+    );
+  }, [potionLevel, timeReduction, ore, itemMastery, masteryPool]);
+
+  return (
+    <>
+      <Card>
+        <h2>Extras</h2>
+        <Row>
+          <p>Potion Level</p>
+          <ReactDropdown
+            options={["0", "1", "2", "3", "4"]}
```

Comment:
I mapped the potionSpeed keys so that it increments based on the options while still being 1 to `length of potionSpeeds`. I find that more intuitive than having a dropdown with 1, 1.1, 1.2, etc. as that is interpreted values (as in "read potion description and created a map based on probability") which I find is nice to abstract.
What do you think?